### DISCUSSION TITLE
Compatibility with CouchDB

### DISCUSSION THREAD

**Discussion by Konst80** [cofu-app/cbl-dart#497] [General]
Hello everyone, 

I have a quick question: Can the cbl-dart library be combined with a CouchDB server? Or is this library only restricted to couchabse server?

Thanks in advance.

**Comment by blaugold** [comment]
Hey @Konst80, Couchbase Lite does not support replication with CouchDB, only with Couchbase Server through a Couchbase Sync Gateway.
